openapi.yaml

```diff
@@ -7423,6 +7448,7 @@ paths:
             --type g6-standard-2 \
             --label cloned-linode \
             --backups_enabled true \
+            --placement_group '{"id": 528, "compliant_only": true}' \
```

Comment:
No support for "compliant_only" yet. Need to change this to:

`--placement_group.id 528 \`